### PR TITLE
[doc] PR for Review and Fix Errors introduced by PR #12756 "Backup and restore documentation cleanup"

### PR DESCRIPTION
**Pull request by lizayugabyte**
Note to reviewers: please check commands in the following files:

1. [/manage/backup-restore/snapshot-ysql/](https://deploy-preview-12864--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/snapshot-ysql/)
2. [/manage/backup-restore/snapshot-ycql/](https://deploy-preview-12864--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/snapshots-ycql/)

### CODE REVIEW

**Review by ddhodge — APPROVED** [review]
Minor comments

File: docs/content/preview/manage/backup-restore/export-import-data.md [hunk 1]
```diff
@@ -31,74 +31,62 @@ showAsideToc: true
   </li>
 </ul>
 
-YugabyteDB comes with two utilities – `ysql_dump` and `ysql_dumpall` – that provide functionality to export the data into a SQL script. These utilities derive from PostgreSQL `pg_dump` and `pg_dumpall`.
+YugabyteDB includes `ysql_dump` and `ysql_dumpall` utilities that allow you to export data into a SQL script. These utilities derive from PostgreSQL `pg_dump` and `pg_dumpall`.
 
-In general, you should use other ways to take backups, such as the [distributed backup and recovery](../snapshot-ysql/) capability. However, it makes sense to use `ysql_dump` and `ysql_dumpall` if you intend to restore the data on a database other than YugabyteDB, or if having data in SQL format is a requirement for other reasons, such as regulations.
+In general, it is recommended use other means for backups, such as [distributed backup and recovery](../snapshot-ysql/). However, you can use `ysql_dump` and `ysql_dumpall` if you intend to restore the data on a database other than YugabyteDB, or if having data in SQL format is a requirement for other reasons, such as regulations.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/export-import-data.md:36]: fixed

File: docs/content/preview/manage/backup-restore/export-import-data.md [hunk 2]
```diff
@@ -31,74 +31,62 @@ showAsideToc: true
...
+./postgres/bin/ysql_dumpall > <file>
 ```
 
-Where `<file>` is the path to the resulting SQL script file.
+*file* is the path to the resulting SQL script file.
 
-Two of the script's common command-line options are:
+The following are two of the script's common command-line options:
 
 * `--roles-only` exports just the roles.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/export-import-data.md:71]: fixed

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 3]
```diff
@@ -16,75 +16,61 @@ isTocNested: true
...
-By default, the history retention period is controlled by the [history retention interval flag](../../../reference/configuration/yb-tserver/#timestamp-history-retention-interval-sec). This flag is applied cluster-wide, to every YSQL database and YCQL keyspace.
-
-However, when [PITR is enabled](#creating-a-schedule) for a database or a keyspace, YugabyteDB adjusts the history retention for that database/keyspace based on the interval between the snapshots. You're _not_ required to manually set the cluster-wide flag in order to use PITR.
+* Each snapshot is kept for three days.
 
-{{< /note >}}
+By default, the history retention period is controlled by the [history retention interval flag ](../../../reference/configuration/yb-tserver/#timestamp-history-retention-interval-sec)applied cluster-wide to every YSQL database and YCQL keyspace.
 
-{{< note title="Space consumption" >}}
+However, when [PITR is enabled](#creating-a-schedule) for a database or a keyspace, YugabyteDB adjusts the history retention for that database or keyspace based on the interval between the snapshots. You are not required to manually set the cluster-wide flag in order to use PITR.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/point-in-time-recovery.md:40]: fixed

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 4]
```diff
@@ -16,75 +16,61 @@ isTocNested: true
...
 
-There are no technical limitations on the retention target. However, it's important to keep in mind that by increasing the number of snapshots stored, you also increase the amount of space required for the database. The actual overhead depends on the workload, so we recommend to estimate it by running tests based on your applications.
+There are no technical limitations on the retention target. However, when you increase the number of stored snapshots, you also increase the amount of space required for the database. The actual overhead depends on the workload, therefore it is recommended to estimate it by running tests based on your applications.
 
-{{< /note >}}
+The preceding sample configuration ensures that at any moment there is a continuous change history maintained for the last three days. When you trigger a restore, YugabyteDB selects the closest snapshot to the timestamp you provide, and then use flashback in that snapshot.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/point-in-time-recovery.md:44]: fixed

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 5]
```diff
@@ -16,75 +16,61 @@ isTocNested: true
...
-Assuming the retention target is 3 days, you should create a schedule that creates a snapshot once a day (every 1,440 minutes), and retains it for 3 days (4,320 minutes):
+Assuming the retention target is three days, you can execute the following command to create a schedule that produces a snapshot once a day (every 1,440 minutes) and retains it for three days (4,320 minutes):
 
 ```sh
-./bin/yb-admin create_snapshot_schedule 1440 4320 my_database
+./bin/yb-admin create_snapshot_schedule 1440 4320 <database_name>
 ```
 
-Once completed, the command will return and print out the unique ID of the newly created snapshot schedule:
+The followig output is a unique ID of the newly-created snapshot schedule:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/point-in-time-recovery.md:73]: fixed

File: docs/content/preview/manage/backup-restore/snapshot-ysql.md [hunk 6]
```diff
@@ -33,152 +33,146 @@ showAsideToc: true
...
 
-```
+```output
 Started snapshot creation: 0d4b4935-2c95-4523-95ab-9ead1e95e794
 ```
 
 You can then use this ID to check the status of the snapshot, [delete it](#delete-a-snapshot), or use it to [restore the database](#restore-a-snapshot).
 
-The `create_database_snapshot` command exits immediately, but the snapshot may take some time to complete. Before using the snapshot, verify its status with the [`list_snapshots`](../../../admin/yb-admin/#list-snapshots) command:
+The `create_database_snapshot` command exits immediately, but the snapshot may take some time to complete. Before using the snapshot, verify its status with by executing the [`list_snapshots`](../../../admin/yb-admin/#list-snapshots) command, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/manage/backup-restore/snapshot-ysql.md:58]: fixed

**Comment by lizayugabyte** [comment]
@jasonriddell I'm going to merge this PR, because otherwise we have commands that don't run in our customer-facing docs.
This still needs a review.

**Comment by jasonriddell** [comment]
Hi Liza,

Apologies for the delay, the following needs to be changed for [/manage/backup-restore/snapshot-ysql/](https://deploy-preview-12864--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/snapshot-ysql/) :

`./bin/yb-admin delete_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> delete_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794`

`./bin/yb-admin restore_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> restore_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794`

`./bin/yb-admin import_snapshot <database_name>.snapshot <database_name>`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> import_snapshot <database_name>.snapshot <database_name>`

All of these default to localhost:7100 but our default configuration doesn't listed on localhost:7100 (hence the reason why all the other commands specify the -master_addresses.

I am not a ycql expert, but I believe that all of these commands need changing on this page [/manage/backup-restore/snapshot-ycql/](https://deploy-preview-12864--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/snapshots-ycql/) :

`./bin/yb-admin create_keyspace_snapshot my_keyspace`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> create_keyspace_snapshot my_keyspace`

`./bin/yb-admin create_snapshot my_keyspace my_table`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> create_snapshot my_keyspace my_table`

`./bin/yb-admin delete_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e`
to
./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> delete_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e

`./bin/yb-admin list_snapshots`
to
./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> list_snapshots

`./bin/yb-admin restore_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> restore_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e`

`./bin/yb-admin export_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e my_keyspace.snapshot`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> export_snapshot a9442525-c7a2-42c8-8d2e-658060028f0e my_keyspace.snapshot`

`./bin/yb-admin import_snapshot my_keyspace.snapshot my_keyspace`
to
`./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> import_snapshot my_keyspace.snapshot my_keyspace`

if you don't have the -master_addresses and aren't listening on localhost:7100 (like all of our yb-anywhere installations) you will get the following timeout when trying to run the command:

```
[yugabyte@ip-172-158-30-165 ~]$ ./master/bin/yb-admin import_snapshot
Timed out (yb/rpc/rpc.cc:224): Unable to establish connection to leader master at [localhost:7100]. Please verify the addresses.

: Could not locate the leader master: GetLeaderMasterRpc(addrs: [localhost:7100], num_attempts: 345) passed its deadline 990348.679s (passed: 60.313s): Network error (yb/util/net/socket.cc:540): recvmsg error: Connection refused (system error 111)
```

**Pull request merged**